Queue/Backend/Sentinel.php

```diff
@@ -36,6 +41,9 @@ protected function connect()
                 $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false);
```

Comment:
@snake14 I don't have redis sentinel installed, so couldn't verify it, shouldn't we pass the password here ?
```suggestion

                $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false, $db = 0, $this->usePasswordForSentinelInstances && !empty($this->password) ? $this->password: null);
```

Reply:
@AltamashShaikh No. That was causing issues. [Lines 44-46](https://github.com/matomo-org/plugin-QueuedTracking/pull/300/files#diff-9dc84d965136dd2f5e8d0b5a630f946f3ae17e2f27d5de8ffc489b75e70186f8R44-R46) handle the authentication if configured.

Reply:
Okay :+1: